cpp/unreal_plugins/SpCore/Source/SpCore/Unreal.h

```diff
@@ -0,0 +1,423 @@
+//
+// Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+//
+
+#pragma once
+
+#include <concepts>    // std::derived_from
+#include <cstring>     // std::memcpy
+#include <map>
+#include <ranges>      // std::views::filter, std::views::transform
+#include <string>
+#include <type_traits> // std::is_base_of
+#include <vector>
+
+#include <Components/ActorComponent.h>
+#include <Components/SceneComponent.h>
+#include <Containers/Array.h>        // TArray
+#include <Containers/StringConv.h>   // TCHAR_TO_UTF8, UTF8_TO_TCHAR
+#include <Containers/UnrealString.h> // FString::operator*
+#include <EngineUtils.h>             // TActorIterator
+#include <GameFramework/Actor.h>
+#include <HAL/Platform.h>            // TCHAR, TEXT
+#include <UObject/NameTypes.h>       // FName
```

Comment:
Let's align these comments.

Reply:
The rule I've been following so far is to align within each include block as far left as possible, but not to align across blocks.

Reply:
Confirmed.